### ISSUE TITLE
ChromeCast reconnect

### ISSUE THREAD

**Issue by kuumaur** [philippe44/LMS-to-Cast#24]
Hi, thank you for all the work and time you invested in this project.
I love it. I have just added 3 ChromeCast devices to my home.
Since all is running good, I have one issue. Well I guess its my fault, but I need some explanation or help.
This link and discussion is hard to follow ( https://forums.slimdevices.com/showthread.php?104614-Announce-CastBridge-integrate-Chromecast-players-with-LMS-(squeeze2cast)&highlight=chromecast+reconnect ) and I could not find a solution.
Here is my description of my Issue:
I see all the Devices as Player in LMS Web Interface. They just do not play/react.
When I restart the LMS service, the ChromeCast Players are working like a Charm, until I switch them off (by power cut/switch).
After power them on again, I can see the Chromecast devices as players, but until restart of the LMS Service they do nothing.
What do I wrong? What did I misconfigure?

The LMS Service is running on a Debian Linux System, inside a LXC Container.
Logitech Media Server Version: 8.1.1 - 1610364019 @ Thu Jan 14 06:22:21 CET 2021
Hostname: lms-prod
Server IP Address: 192.168.200.109
Server HTTP Port Number: 9000
Operating system: Debian - EN - utf8
Platform Architecture: x86_64-linux
Perl Version: 5.32.1 - x86_64-linux-gnu-thread-multi
Audio::Scan: 1.02
IO::Socket::SSL: 2.069
Database Version: DBD::SQLite 1.58 (sqlite 3.22.0)

uname=Linux lms-prod 5.13.19-2-pve #1 SMP PVE 5.13.19-4 (Mon, 29 Nov 2021 12:10:09 +0100) x86_64 GNU/Linux
Debian 11
ChromaCast Bridge v.1.81.0

**Comment by philippe44** [comment]
I'd need to see a log and copy of your config file. Maybe best is to move to LMS' forum

**Comment by kuumaur** [comment]
Hi, I posted this Issue here:
https://forums.slimdevices.com/showthread.php?104614-Announce-CastBridge-integrate-Chromecast-players-with-LMS-(squeeze2cast)&p=1042149#post1042149

Thank you.

**Comment by freggal** [comment]
Hello
I had the same problem. As suggested, disabled IGMP Snooping in my LAN switch, and that fixed it.
Thanks for your work !

**Comment by philippe44** [comment]
I've uploaded new version (int the dev channel) that should handle the IGMP snooping issue
